platform/linux-generic/odp_schedule.c

```diff
@@ -163,6 +163,7 @@ typedef struct {
 		int stash_num; /**< Number of stashed enqueue operations */
 		uint8_t in_order; /**< Order status */
 		lock_called_t lock_called; /**< States of ordered locks */
+		uint32_t lock_index;
```

Comment:
This should have a doxygen description for consistency with surrounding vars.

Reply:
Done